spec/requests/campaigns_request_spec.rb

```diff
@@ -171,6 +181,35 @@
             expect(campaign.valid).to eq true
           end
         end
+
+        context 'with valid parameters and project_id' do
```

Comment:
we don't have a case that tests creating with both a seller and a project

Reply:
added!